scripts/globals/transport.lua

```diff
@@ -33,4 +60,15 @@ tpz.transport.captainMessage = function(npc, triggerID, messages)
     for _, player in pairs(playersInZone) do
         player:showText(player, messages[triggerID])
     end
+end
+
+tpz.transport.dockPorterMhauraMessages = function(npc, triggerID, messages)
```

Comment:
Not going to force you to globalize this _juuusstt_ yet, but you can save yourself a little effort later by:
```lua
tpz.transport.messages = {}
tpz.transport.messages.mhaura = function(npc, triggerID, messages)
```

(Later we can extend this to instead be just one function which takes a `dock`, and then pulls the appropriate tables based on the dock, but I realize right now that the focus is to fix the ferry)